server/flaskr/database/postgres/handlers/user_data_handler.py

```
@@ -9,24 +11,47 @@ def create_user(cls, firstName: str, lastName: str, email: str, passwordHash: st
+                )
+
+                query = f"INSERT INTO usersRoles (userId, roleId) VALUES (%s, %s);"
+                cur.execute(query, (userId, 1))
+
+                return True
+        except Exception as e:
+            pass
+        return False
+
+    @classmethod
+    def get_users(cls, orgId: int):
```

Ruff (0.15.15)

[warning] Missing return type annotation for classmethod `get_users`

(ANN206)